README.md

```diff
@@ -163,9 +163,6 @@ The following feature set is not considered stable yet, but we are committed to
 
 #### Anything under the `COSIGN_EXPERIMENTAL` environment variable
 
-* Integration with the `Rekor` transparency log
```

Comment:
should we just remove the COSIGN_EXPERIMENTAL var reference? I think it's not used at all

Reply:
I'm fine with removing it, but my thinking was that we may want to keep it in case we release other features as experimental in the future, so precedent has been set.